client/vscode/src/chatActivation.ts

```diff
@@ -54,26 +55,40 @@ export function registerChat(languageClient: LanguageClient, extensionUri: Uri,
         languageClient.info(`vscode client: Received ${JSON.stringify(message)} from chat`)
 
         switch (message.command) {
+            case END_CHAT_REQUEST_METHOD:
```

Comment:
i would expect this case to be already taken care of by the 'default' case, I think we can remove this specific case